corehq/apps/domain/decorators.py

```diff
@@ -341,17 +341,17 @@ def login_or_api_key_ex(allow_cc_users=False, allow_sessions=True, require_domai
     )
 
 
-def login_or_oauth2_ex(allow_cc_users=False, allow_sessions=True, require_domain=True):
+def login_or_oauth2_ex(allow_cc_users=False, allow_sessions=True, require_domain=True, oauth_scopes=['access_apis']):
```

Comment:
nit: mutable default argument. should probably be replaced with a default of `None` and then the default added as the first line of the function:

```
oauth_scopes = oauth_scopes or ['access_apis']
```

Reply:
Ah, that's an easy mistake. Thanks for pointing out.